### PR TITLE
fix: Improve ARIA labelling fold controls

### PR DESCRIPTION
**Pull request by akoreman**
*Description of changes:* For a11y compliance, buttons that show/hide content should have an `aria-expanded` state. This adds that state and changes the aria label accordingly.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by oykuyilmaz — APPROVED** [review]

File: demo/i18n.html [hunk 1]
```diff
@@ -72,9 +72,9 @@
         "Autocomplete suggestions": "Предложения автозаполнения",
                 
         "Cursor at row $0": "",
-        "Unfold rows $0 to $1": "",
+        "Code folding, rows $0 through $1": "",
+        "Code folding, row $0": "",
         "Unfold code": "",
-        "Fold at row $0": "",
```
Comment (InspiredGuy) [inline demo/i18n.html:77]: if someone was using the old translation keys, what is going to happen? can we at least show a warning? can we support both by deprecating the old one?

Comment (akoreman) [reply demo/i18n.html:77]: added a warning when a Ace tries to replace a string using the `nls` function but can't find a translation in the provided messages.

Comment (andredcoliveira) [reply demo/i18n.html:77]: Does this mean this is a breaking change? If so, can we avoid it or is it intentional?

**Review by whazor — APPROVED** [review]
looks nice!

**Review by InspiredGuy — APPROVED** [review]

**Pull request merged**